.changeset/five-numbers-buy.md

```diff
@@ -0,0 +1,6 @@
+---
+'@nordeck/matrix-neoboard-widget': patch
+'@nordeck/matrix-neoboard-react-sdk': patch
+---
+
+bump dev dependencies
```

Comment:
We usually don't put dependency updated to the changelog.

Reply:
done